plugins/callback/logstash.py

```diff
@@ -19,16 +20,25 @@
         description: Address of the Logstash server
         env:
           - name: LOGSTASH_SERVER
+        ini:
+          - section: callback_logstash
+            key: server
```

Comment:
```suggestion
            key: server
            version_added: 1.0.0
```
Same for the other new options.

Reply:
@felixfontein version_added is needed to all key in section ini ?

Reply:
Only for the ones you added (which in this case seems to be all of them).